spring-integration-core/src/main/java/org/springframework/integration/support/json/EmbeddedJsonHeadersMessageMapper.java

```diff
@@ -135,7 +138,16 @@ public void setRawBytes(boolean rawBytes) {
 		this.rawBytes = rawBytes;
 	}
 
-	public Collection<Pattern> getHeaderPatterns() {
+	/**
+	 * Set to true to make the header name pattern match case sensitive.
+	 * Default false.
+	 * @param caseInsensitive true to make case sensitive.
```

Comment:
Merging after fixing this typo.
We definitely can address docs in the separate PR and even during RC-GA phase.